pkg/odo/cli/dev/dev.go

```diff
@@ -268,11 +268,11 @@ func regenerateComponentAdapterFromWatchParams(parameters watch.WatchParameters)
 }
 
 func (o *DevOptions) HandleSignal() error {
+	fmt.Fprintf(o.out, "\n\nCancelling deployment. Press Ctrl-c again to terminate immediately\n\n")
```

Comment:
The message is not accurate. If you press Ctrl-C again, nothing happens (precisely because we didn't return from the signal handler)

Reply:
Removed. It wasn't valid to add this without adding the other parts of your example commit.

Reply:
You could keep the first part of the message. It is disappoointing for the user to press Ctrl-C and see nothing happening during the deployment time

Reply:
WDYT about a message like below:
```go
fmt.Fprintf(o.out, "\n\nCancelling deployment.\n\nThis is non-preemptive operation, it will wait for other tasks to finish first")
```